plugins/services/src/js/components/forms/MultiContainerVolumesFormSection.js

```diff
@@ -74,6 +76,26 @@ class MultiContainerVolumesFormSection extends Component {
     });
   }
 
+  getUnknownVolumeConfig(volumes, key, offset) {
```

Comment:
Wanted to trigger that view with the app config but instead I'm getting an empty box

![screen shot 2018-01-09 at 16 13 24](https://user-images.githubusercontent.com/186223/34727716-0be769da-f558-11e7-946e-bc15f2a3795e.png)


```
{
  "id": "/asd",
  "containers": [
    {
      "name": "container-1",
      "resources": {
        "cpus": 0.1,
        "mem": 128
      },
      "image": {
        "id": "alpine",
        "kind": "DOCKER"
      },
      "volumeMounts": [
      ]
    }
  ],
  "scaling": {
    "instances": 1,
    "kind": "fixed"
  },
  "networks": [
    {
      "mode": "host"
    }
  ],
  "volumes": [
    {
      "name": "asdf",
      "unknownPath": "sdf"
    }
  ],
  "fetch": [],
  "scheduling": {
    "placement": {
      "constraints": []
    }
  }
}
```

Reply:
This one is fixed.